app/client/views/application/application.html

```diff
@@ -78,21 +78,24 @@
 
         <div class="field">
           <label>
-              What would you like to learn or get out of HackMIT? (optional)
+              What would you like to learn or get out of HackUCI?
           </label>
-          <textarea ng-model="user.profile.essay">
+          <textarea ng-model="user.profile.essay"
+            name="essay"
+            placeholder="Please enter your response here (100 chars min, 1500 max)">
           </textarea>
+          <span>{{user.profile.essay.length}}/1500</span>
```

Comment:
@af4ro could you align the character count to the right and add an error message on the left in red if it's <100 or >1500?

Reply:
@af4ro also could you add real time input validation too so the box will turn red + error message will be displayed when it's <100 or >1500?